packages/vaex-ml/vaex/ml/sklearn.py

```diff
@@ -95,3 +91,24 @@ def fit(self, df, target, **kwargs):
         X = df[self.features].values
         y = df.evaluate(target)
         self.model.fit(X=X, y=y, **kwargs)
+
+    def fit_transform(self, df, target, **kwargs):
```

Comment:
I don't think it makes sense to have this in this class, I'd say some base class in vaex.ml should do this so all transformers have it? (in fact we do, ml/transformers.py::Transformer.